prow/plugins/label/label.go

```diff
@@ -287,21 +287,31 @@ func handle(gc githubClient, log *logrus.Entry, ae assignEvent, sc slackClient,
 		toRepeat = ae.getRepeats(sigMatches, existingLabels)
 	}
 
-	if len(toRepeat) > 0 {
-		if !isMember {
-			msg := fmt.Sprintf(chatBack, strings.Join(toRepeat, ", "))
-			if err := gc.CreateComment(ae.org, ae.repo, ae.number, plugins.FormatResponseRaw(ae.body, ae.url, ae.login, msg)); err != nil {
-				log.WithError(err).Errorf("Could not create comment \"%s\".", msg)
-			}
+	if len(toRepeat) > 0 && !isMember {
+		msg := fmt.Sprintf(chatBack, strings.Join(toRepeat, ", "))
+		if err := gc.CreateComment(ae.org, ae.repo, ae.number, plugins.FormatResponseRaw(ae.body, ae.url, ae.login, msg)); err != nil {
+			log.WithError(err).Errorf("Could not create comment \"%s\".", msg)
 		}
+	}
 
-		// If sig matches then send a notification on slack.
-		for _, sig := range sigMatches {
-			msg := fmt.Sprintf("Message: ```%s```\nIssue: %d, %s\nUrl: %s.", ae.body, ae.issue.Number, ae.issue.Title, ae.issue.HTMLURL)
-			if err := sc.WriteMessage(plugins.FormatResponseRaw(ae.body, ae.url, ae.login, msg), "sig-"+sig[1]); err != nil {
-				log.WithError(err).Error("Failed to send message on slack channel: ", "sig-"+sig[1], " with message ", msg)
+	for _, match := range sigMatches {
```

Comment:
We're going to let non-members ping slack?

Reply:
We already reiterate mentions from non-members on github so I figured we would want to allow non-members to ping slack as well. I could easily change this to only message slack for comments left by org members if that is preferred though.